src/utils/aspect-ratio-support.tsx

```diff
@@ -67,6 +61,12 @@ export const AspectRatioContainer = (props: AspectRatioContainerProps): JSX.Elem
         width = props.width || '100%';
     }
 
+    // if width and height were computed, we ignore the aspectRatio.
+    // if aspect ratio is 0, we use the original image proportions.
+    const withCssAspectRatio = (width === undefined || height === undefined) && props.aspectRatio !== 0;
```

Comment:
Aspect ratio should be ignored if we were able to calculate width/height (even if they are provided as strings, like percentages or pixels).